kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/SectionSidePanel/SectionEditor.vue

```diff
@@ -461,7 +455,6 @@
     padding: 1em;
     margin-top: 1em;
     background-color: #ffffff;
-    border-top: 1px solid black;
```

Comment:
While we don't want the solid black border here, the figma shows a dropshadow, so we should replace with that, rather than just remove the border 

<img width="636" alt="Screenshot 2025-04-10 at 2 04 56 PM" src="https://github.com/user-attachments/assets/b9b4a221-72a9-4225-b257-3c5718c83397" />